config/webpack.config.js

```diff
@@ -145,7 +145,7 @@ module.exports = env => {
           })
         },
         {
-          test: /(\.png|\.gif)$/,
+          test: /(\.png|\.gif|\.svg)$/,
```

Comment:
Perhaps it should be shorted:
```suggestion
          test: /\.(png|gif|svg)$/,
```